src/util/privacy.ts

```diff
@@ -4,24 +4,41 @@ import { getConfig } from 'sqlite/config';
 
 const DEFAULT_RADIUS = 200;
 let privateZones: KDBush | undefined;
+const privateZoneWithRadius: [number, number, number][] = [];
 
-export const isPrivateLocation = async (lat: number, lon: number): Promise<boolean> => {
+export const isPrivateLocation = async (
+  lat: number,
+  lon: number,
+): Promise<boolean> => {
   if (!lat || !lon) {
     return false;
   }
   const { privacyRadius } = await getConfig('privacyRadius');
   const { x, y } = latLonToWebMercator(lat, lon);
   const radius = privacyRadius || DEFAULT_RADIUS;
-  return privateZones ? !!privateZones.within(x, y, radius * 1.25).length : false;
-}
+  return privateZones
+    ? !!privateZones.within(x, y, radius * 1.25).length
```

Comment:
@HariHive 
So new radius for every privacy zone is not getting there, if I get it right?
Meaning `isPrivateLocation` will use the default radius instead of using the one for particular private zone